### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aswarren**
Pull request to help fix currently broken host RNA-Seq support.

### CODE REVIEW

File: public/js/p3/widget/GenomeNameSelector.js [hunk 1]
```diff
@@ -3,48 +3,48 @@ define([
...
     searchAttr: 'genome_name',
     extraSearch: ['genome_id'],
     queryExpr: '*${0}*',
     queryFilter: '',
-    resultFields: ['genome_id', 'genome_name', 'strain', 'public', 'owner', 'reference_genome'],
+    resultFields: ['genome_id', 'genome_name', 'strain', 'public', 'owner', 'reference_genome', 'taxon_id'],
     includePrivate: true,
     includeOtherPublic: true,
     referenceOnly: true,
+    // TODO: make this defaulted to false
```
Comment (hyoo) [inline public/js/p3/widget/GenomeNameSelector.js:28]: Do we need to change the defaulted value to False before we merge?

Comment (aswarren) [reply public/js/p3/widget/GenomeNameSelector.js]: @hyoo The comment itself is outdated. Fixing. Thanks.

**Comment by nconrad** [comment]
Note: requires https://github.com/PATRIC3/p3_api/pull/71

**Comment by hyoo** [comment]
@nconrad good catch. i will merge and deploy it first. Do you see any other issues?

**Pull request merged**